relayer-cli/src/commands/query/connection.rs

```diff
@@ -98,11 +98,89 @@ impl Runnable for QueryConnectionEndCmd {
     }
 }
 
+/// Command for querying the channel identifiers associated with a connection.
+/// Sample invocation:
+/// `cargo run --bin relayer -- -c simple_config.toml query connection channels ibc-0 connection-0`
+#[derive(Clone, Command, Debug, Options)]
+pub struct QueryConnectionChannelsCmd {
+    #[options(free, help = "identifier of the chain to query")]
+    chain_id: Option<ChainId>,
+
+    #[options(free, help = "identifier of the connection to query")]
+    connection_id: Option<String>,
+}
+
+#[derive(Debug)]
+struct QueryConnectionChannelsOptions {
+    connection_id: ConnectionId,
+}
+
+impl QueryConnectionChannelsCmd {
+    fn validate_options(
+        &self,
+        config: &Config,
+    ) -> Result<(ChainConfig, QueryConnectionChannelsOptions), String> {
+        let chain_id = self
+            .chain_id
+            .clone()
+            .ok_or_else(|| "no chain chain identifier provided".to_string())?;
+        let chain_config = config
+            .chains
+            .iter()
+            .find(|c| c.id == chain_id)
```

Comment:
This could use the `Config::find_chain` method introduced recently in #510.

Reply:
Good point, thanks for bringing that up! 

ea129d390229e4e3fe0403c880174f33e38d1e5e